index.js

```diff
@@ -81,7 +81,7 @@ class Transaction {
     }, {
       name: 'v',
       allowZero: true,
-      default: new Buffer([0x1c])
+      default: ((data.chainId) ? data.chainId : new Buffer([0x01]))
```

Comment:
Can be simplified into `data.chainId || (new Buffer([0x01]))`

Not sure whether 0x01 or 0x1c is correct though.

Reply:
True.
0x01 is correct because ```v``` replaced by ```CHAIN_ID``` according to [EIP-155](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md). You can see their example.

Reply:
"`CHAIN_ID`: 1" in [EIP-155](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md) is, I believe, a parameter meant to understand the content of the EIP. It does not indicate that raw, unsigned, transactions should default the chainId to 1.